Mathlib/CategoryTheory/Category/Quiv/Shapes.lean

```diff
@@ -0,0 +1,181 @@
+import Mathlib.CategoryTheory.Category.Quiv
+import Mathlib.CategoryTheory.Limits.Shapes.Terminal
+import Mathlib.CategoryTheory.Generator.Basic
+
+namespace CategoryTheory.Quiv
+universe v u v₁ u₁ v₂ u₂
+open Limits
+
+def Empty : Quiv.{v, u} := ⟨PEmpty, ⟨fun _ _ ↦ PEmpty⟩⟩
```

Comment:
There are a few issues with this PR. One of these is that in category theory (and the same should apply for quivers and refl quivers), it is better practice not to define directly objects in `Cat`, but rather define a `Category` instance on a type, and only later, we may use `Cat.of C` if we need a term in `Cat`.
As it was observed by @robin-carlier , some of the quivers considered in this PR are refl quivers or even categories (and may have already been defined elsewhere), which also supports my previous remark about `Quiver` vs `Quiv`.